static/js/utils.js

```diff
@@ -102,6 +102,34 @@ function formatField(value){
   }
 }
 
+function chunkTextAsHtml(text, chunkSize){
+    // Chunks up a large string to be used in an html rendered label. Chunksize ~150 works pretty well on my vertical
+    // monitor so it should work for most monitors.
```

Comment:
Maybe the comment about the choice of chunk size can go where the chunk size is set